src/style.css

```diff
@@ -1 +1,67 @@
 /* Styles go here */
+body {
+  margin: 0 auto;
+  text-align: center;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 50px;
+  padding-right: 50px;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-family: "Roboto", sans-serif;
```

Comment:
Better move `font-family` to body, usually we have one font per page, no need to specify for individual tags